aries-site/src/examples/templates/forms/SettingsExample.js

```diff
@@ -77,7 +77,7 @@ export const SettingsExample = () => {
               </FormField>
             </Box>
             <Box margin={{ bottom: 'small' }}>
-              <Text color="text-weak" size="xsmall">
+              <Text color="text" size="xsmall" weight={600}>
```

Comment:
This input is a CheckBox. With the recent CheckBox work, I believe we can eliminate this Text, while relying on FormField label + moving CheckBox to be a child rather than specified as a prop.

Reply:
Great, let me look into that.

Reply:
Made some changes to implement this. FormField will need a bit of discussion on the Grommet side to look at how labels are handled when Checkbox is the input -- leads to a situation where multiple labels are present for a single input since Checkbox and FormField need label props.